src/index.html

```diff
@@ -164,7 +164,7 @@
                 <div>
                     <p>
                         <!--<img src="img/Receive.png" alt="" width="48" height="48"> &nbsp;Receive your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:-->
-                        Download your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:
+                        <!--Download your <a href="files/eva_submission_package.zip" target="_blank" onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'eva_submission_package.zip'});">submission pack</a>, which will include:-->

```

Comment:
Could the submission template changes be extracted into another PR? They don't relate to Google Analytics, but affect the submission process quite strongly.